Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptWrapperAddController.java

```diff
@@ -83,9 +84,13 @@ private void initBinder(WebDataBinder binder) {
      */
     @RequestMapping(value = "auth/conceptlist/addconceptwrapper")
     public String prepareConceptWrapperAdd(@ModelAttribute ConceptWrapperAddBean conceptWrapperAddBean,
-            HttpServletRequest req, ModelMap model) {
+            HttpServletRequest req, ModelMap model) throws LuceneException, IllegalAccessException, IndexerRunningException{
         model.addAttribute("types", conceptWrapperService.fetchAllConceptTypes());
         model.addAttribute("lists", conceptWrapperService.fetchAllConceptLists());
+        String wrapperId = conceptWrapperAddBean.getWrapperids();
+        ConceptEntry entry = conceptManager.getWordnetConceptEntry(wrapperId);
```

Comment:
why is the id called 'wrapperId'? It's the id of a concept, not of a wrapper, isn't it?

Reply:
It is concept id only. That was the name given in the home.jsp file to pass the id to controller. So, I used the same id name(wrapperId) in the controller. It is in fact the id of the concept.